lib/src/http_request_impl.dart

```diff
@@ -3,14 +3,19 @@ import 'package:meilisearch/src/version.dart';
 import 'http_request.dart';
 import 'exception.dart';
 
+const bool _kIsWeb = bool.fromEnvironment('dart.library.js_util');
+
 class HttpRequestImpl implements HttpRequest {
   HttpRequestImpl(this.serverUrl, this.apiKey, [this.connectTimeout])
       : dio = Dio(
           BaseOptions(
             baseUrl: serverUrl,
             headers: <String, Object>{
               if (apiKey != null) 'Authorization': 'Bearer $apiKey',
-              'User-Agent': Version.qualifiedVersion,
+              if (_kIsWeb)
+                'X-Meilisearch-Client': Version.qualifiedVersionWeb
+              else
+                'User-Agent': Version.qualifiedVersion,
```

Comment:
```suggestion
                'X-Meilisearch-Client': [Version.qualifiedVersion, if (_kIsWeb) Version.qualifiedVersionWeb].join(',')
```

Reply:
this will result in `Meilisearch Dart Web (v1.0.0),Meilisearch Dart (v1.0.0)` missing the `[` and `]`, is this OK ?

Reply:
yeah!